features/get_task.feature

```diff
@@ -2,4 +2,7 @@ Feature: Get a task
 
   Scenario: User gets a task
     TODO: Add steps for getting a task
-    
\ No newline at end of file
+  Given the user is using the task manager application for looking at or getting a task by name
+  And at least one task is available in the list 
+  When the user gets a task with a specific task_name "My Task"
```

Comment:
When the user requests the task with the specific task name: "My Task" and the description "My Description"